src/User/resources/views/admin/_user.php

```diff
@@ -13,8 +13,60 @@
  * @var yii\widgets\ActiveForm $form
  * @var \Da\User\Model\User $user
  */
+
+use Da\User\Dictionary\UserSourceType;
+use kartik\typeahead\Typeahead;
```

Comment:
This is not a requirement of the module, projects without this module installed will receive an error. Must be added in `composer.json`. However, considering the restricted usage of this change, I'd rather avoid adding as a global requirement. Maybe we can avoid adding it to `composer.json`, we could make a library check during usuario's bootstrap. If the search flag is enabled, check if this class is defined.